indico/modules/events/papers/controllers/display.py

```diff
@@ -159,3 +162,20 @@ class RHDeletePaperComment(RHPaperCommentBase):
     def _process(self):
         delete_comment(self.comment)
         return jsonify_data(flash=False)
+
+
+class RHDisplayReviewingArea(RHPapersBase):
+    def _checkProtection(self):
+        if not session.user:
+            raise Forbidden
+        if session.user not in self.event_new.cfp.layout_reviewers and \
```

Comment:
an `is_reviewer(user, role=None)` method (in the CFP class) might be useful for this. it would check whether the user is a reviewer for any enabled reviewing type (or for the specific one if role is not None and that functionality is needed somewhere)